easybuild/easyblocks/i/icc.py

```diff
@@ -67,6 +68,12 @@ def __init__(self, *args, **kwargs):
 
         self.debuggerpath = None
 
+        if LooseVersion(self.version) >= LooseVersion('2016') and self.cfg['components'] is None:
```

Comment:
`None` is the default value

if you explicitely set `components = []`, it should magically become `components = [ALL]` internally?

instead, you should get an error message that an empty list of components doesn't make sense (that's not there yet, but I'll add it)